monai/networks/nets/vnet.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional, Tuple, Type, Union
+
+import torch
+import torch.nn as nn
+
+from monai.networks.blocks.convolutions import Convolution
+from monai.networks.layers.factories import Act, Conv, Dropout, Norm, split_args
+
+
+def get_acti_layer(act: Optional[Union[Tuple[str, Dict], str]], nchan: int):
```

Comment:
this could be slightly more flexbile:
```python
def get_acti_layer(act: Union[Tuple[str, Dict], str], nchan: int = 0):
    if act == "prelu" and nchan > 0:
...
```
the type `Optional` essentially means allowing for `None` (https://docs.python.org/3/library/typing.html#typing.Optional), but `None` doesn't work here. so please remove `Optional` (there're a few other places in this PR where `Optional` could be removed)

Reply:
Thanks, I've updated.